src/components/Table/sgds-table.ts

```diff
@@ -115,12 +133,67 @@ export class SgdsTable extends SgdsElement {
         })}
         tabindex="0"
       >
-        <table class="table">
-          ${this._renderTable()}
-        </table>
+        <slot id="table-slot" class="table" @slotchange=${this._handleSlotChange}> </slot>
+
+        ${this.tableContents.length === 0
+          ? html`<table class="table">
+              ${this._renderTable()}
+            </table>`
+          : ""}
       </div>
     `;
   }
 }
 
```

Comment:
add jsdocs to document the slot as mentioned above. 

add `@summary` description too